aviatrix/resource_aviatrix_controller_access_allow_list.go

```diff
@@ -0,0 +1,152 @@
+package aviatrix
+
+import (
+	"context"
+
+	"github.com/AviatrixSystems/terraform-provider-aviatrix/v3/goaviatrix"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAviatrixControllerAccessAllowList() *schema.Resource {
+	return &schema.Resource{
+		CreateWithoutTimeout: resourceAviatrixControllerAccessAllowListCreate,
+		ReadWithoutTimeout:   resourceAviatrixControllerAccessAllowListRead,
+		UpdateWithoutTimeout: resourceAviatrixControllerAccessAllowListUpdate,
+		DeleteWithoutTimeout: resourceAviatrixControllerAccessAllowListDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"allow_list": {
+				Type:        schema.TypeList,
```

Comment:
Does the order of this "allow_list" matter? If not, can we use a set instead of a list?